packages/wonder-blocks-tokens/src/util/sizing-utils.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * The baseline for the size tokens.
+ */
+const BASELINE_VALUE = 10;
+
+/**
+ * Converts a rem value to a number (px).
+ * @param value The rem value to convert (includes the unit).
+ * @param includeUnit A boolean for whether to include the string unit.
+ * @param baseline A number to configure the baseline for calculations, e.g. 10.
+ * @returns A string with the px value or a number without.
+ */
+export function remToPx(
+    value: string,
+    includeUnit?: true,
+    baseline?: number,
+): string;
+export function remToPx(
+    value: string,
+    includeUnit: false,
+    baseline?: number,
+): number;
+
+export function remToPx(
```

Comment:
@jandrade I made a change to these CSS utilities to make them easier to use in compute functions. Can you take a look? I used an overload and a param for including the unit, but it could be a different function instead if that's preferred. I figured having one export that's configurable would be nice for ease of use.

Reply:
Update: I simplified this function back to the way it was, since the approach I took wasn't going to work with CSS variables! I kept the function in a separate utils file to make it more portable and testable.